07.bowling_object/lib/bowling.rb

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require_relative './game'
```

Comment:
```suggestion
require_relative 'game'
```

require_relativeを使うのであればこちらがいいと思います〜